### PR TITLE
Feature/adapt to rollups node 1.3.0

### PR DESCRIPTION
**Pull request by endersonmaia**
This PR will adapt to rollups-node:1.3.0.

As steps to simplify our chart code, it will:

- [x] Remove option to install a localnode
- [x] Remove the option to install a Redis instance
- [x] Remove the option to install a PostgreSQL
- [x] Remove the option to pass Redis or PostgreSQL configuration via `.Values`
- [x] Use a single Pod for the cartersi/rollups-node single Go binary
- [x] Adapt to environment variables only approach
- ~[ ] expose /metrics (enable via annotations)~
- [x] expose httpport as a service
- [x] expose ingress for each endpoint (/graphql, /inspect/, /health, ...)
- [x] OPTIONAL: provide `.Values.` for all `cartesi-rollups-node` configuration

### CODE REVIEW

**Comment by oap75** [comment]
> * Remove option to install a localnode
> * [x]  Remove the option to install a Redis instance
> * [x]  Remove the option to install a PostgreSQL
> *

Why do we need to remove these options ? Don't we need to test the helm chart locally with localnode anymore ?

**Comment by endersonmaia** [comment]
> Why do we need to remove these options ? Don't we need to test the helm chart locally with localnode anymore ?

To make it simple, the chart will only be responsible for the rollups-node, everything else should be out of scope.

All the rollups-node dependencies are documented on the cartesi/rollups-node documentation, we could link to it docs.

File: charts/rollups-node/templates/NOTES.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+{{- else if contains "LoadBalancer" .Values.validator.service.type }}
      NOTE: It may take a few minutes for the LoadBalancer IP to be available.
            You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "validator.fullname" . }}'
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "validator.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
-  echo http://$SERVICE_IP:{{ .Values.graphqlServer.service.port }}
-{{- else if contains "ClusterIP" .Values.graphqlServer.service.type }}
+  echo http://$SERVICE_IP:4000/graphql
+{{- else if contains "ClusterIP" .Values.validator.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "validator.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
   export CONTAINER_PORT=$(kubectl get pod --namespace {{ .Release.Namespace }} $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
```
Comment (oap75) [inline charts/rollups-node/templates/NOTES.txt:13]: We do not have `containerPort` in deployment section.

**Review by oap75 — CHANGES_REQUESTED** [review]

**Review by oap75 — CHANGES_REQUESTED** [review]
Need rebase

**Review by oap75 — APPROVED** [review]
LGTM 🚀

**Pull request merged**